### DISCUSSION TITLE
I cannot login my vercel account

### DISCUSSION THREAD

**Discussion by fullstacknator** [vercel/community#4397] [Account Issue]
### Summary

I cannot login to my vercel account and the system says that there is already an account associated with my github email. Is it possible that somebody else could have used my github email?

### Steps to Reproduce

I tried 3 methods to login:
1) click on Login "continue with GitHub" I get:
Log in to Vercel
There is already an account associated with your GitHub email address. Please login with your email and password then link your GitHub account. [Login with Email](https://vercel.com/login/email?next=%252Faccount%252Flogin-connections)
Continue with Email →

1) click on Login "Continue with email" I use this email structure: username@gmail.com
Log in to Vercel
There is no Vercel account associated with this email address. [Sign up?](https://vercel.com/signup)

3) click on Login "Continue with email" I use this email structure: username+vercel@gmail.com
Log in to Vercel
We are unable to complete your signup. 
Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.
← Other Login Options

**Comment by amyegan** [comment]
Are you still getting these errors? If you still aren't able to sign up for a new account, please open a support case using the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Comment by fullstacknator** [comment]
Hi Amy. I still cannot login to my vercel account. I sent email to
***@***.***, I received a case number but it is still in progres and
I cannot deploy. I need to be able to login in order to deploy. I need it
as a matter of urgency. How can I get faster support?

**Reply by amyegan** [reply]
It sounds like you already did the right thing by sending that email to open a case. Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. There are a high number of tickets open right now so wait times are a little longer than normal.

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time
